node_helper.js

```diff
@@ -1,3 +1,4 @@
+var NodeHelper = require("node_helper"); // the absence of which caused of much failure to launch
```

Comment:
```suggestion
var NodeHelper = require("node_helper");
```
Please remove the comment, it does not help to understand the code